internal/usage/usage.go

```diff
@@ -84,6 +84,8 @@ const (
 	NotificationsChannelName        = "Slack channel name. Required for the SLACK notifications type."
 	AlertConfigAPIKey               = "Datadog API Key, Opsgenie API Key, VictorOps API key."
 	APIKey                          = "API Key."
+	ChannelName                     = "The channel name to reconfigure."
+	APIToken                        = "Your Slack API Token."
```

Comment:
let's rename this to `SlackAPIToken` to match the description and avoid someone miss using it

Reply:
I have already tried but I got the same error, I think we shouldn't use `token`